.github/workflows/bench.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           tool: 'cargo'
           output-file-path: output.txt
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          auto-push: ${{ github.event_name == 'push' && github.repository == 'ucan-wg/rs-ucan' && github.ref == 'refs/heads/main' }}
+          auto-push: ${{ github.event_name == 'push' && github.repository == 'ucan-wg/rs_ucan' && github.ref == 'refs/heads/main' }}
```

Comment:
Not sure about this; I did a global find-and-replace to match what I hear is becoming the style, but I'll probably revert this to keep the same name on crates.io

Reply:
Hm. That's interesting. All of the rest of our things are using kebab-case for now:
- `wnfs-wg/rs-wnfs`
- `wnfs-wg/rs-car-mirror`
- `wnfs-wg/rs-skip-ratchet`
- `fission-codes/fission-server`

All the crates.io names are using kebab-case as well.
And of course all of these names are turned into snake_case when importing the module.
The only exception is `skip_ratchet`. That one is also snake_case on crates.io.

Reply:
I think the kebab-case looks nicer because github org names are kebab-case usually, too.

Reply:
> I think the kebab-case looks nicer

Purely aesthetically, I agree 🎨 

In looking around late last year, it looks like a bunch of folks in the Rust community are moving to `snake_case` because it's less confusing than having the `package-name` different from the `crate_name` (`rs-ucan` in GitHub and Cargo.toml but `use rs_ucan` leads to many typos)